### PR TITLE
Documentation language reworked

### PR DESCRIPTION
**Pull request by comradekingu**
Was just reading this. Might pile on some other documents from the documentation.(?)

### CODE REVIEW

**Comment by schlagmichdoch** [comment]
Thanks for revising!

I mean there are more docs: https://github.com/schlagmichdoch/PairDrop/tree/master/docs and the README. Feel free to take a look at those too otherwise I will just merge this as is. Thank you!

**Comment by comradekingu** [comment]
@schlagmichdoch 
Will get to those too in this PR.

File: docs/host-your-own.md [hunk 1]
```diff
@@ -462,28 +503,38 @@ Then, clone the repository and run docker-compose:
...
+
+If you want to test PWA features, you need to trust the CA of the certificate for your local deployment. \
+For your convenience, you can download the crt file from `http://<Your FQDN>:8080/ca.crt`. \
+Install that certificate to the trust store of your operating system. \
+- On Windows, make sure to install it to the `Trusted Root Certification Authorities` store. \
+- On macOS, double-click the installed CA certificate in `Keychain Access`, \
+- expand `Trust`, and select `Always Trust` for SSL. \
+- Firefox uses its own trust store. To install the CA, \
+- point Firefox at `http://<Your FQDN>:8080/ca.crt`. \
+- When prompted, select `Trust this CA to identify websites` and click \"OK\". \
```
Comment (comradekingu) [inline docs/host-your-own.md:531]:
```suggestion
- When prompted, select `Trust this CA to identify websites` and click *OK*. \
```

File: docs/docker-swarm-usage.md [hunk 2]
```diff
@@ -2,42 +2,46 @@
...
+> Specifies the time interval to run the health check. \
+> In this case, the health check is performed every 30 seconds.
 <br>
 
 ```
 --timeout=10s
 ```
-> Specifies the amount of time to wait for a response from the health check command. If the response does not arrive within 10 seconds, the health check will be considered a failure.
-
+> Specifies the amount of time to wait for a response from the \"healthcheck\" command. \
```
Comment (comradekingu) [inline docs/docker-swarm-usage.md:17]: Could be this is "health-check" or something else.(?)

**Comment by comradekingu** [comment]
@schlagmichdoch Ready for review.
While I remember, https://github.com/comradekingu/PairDrop/blob/patch-1/docs/pairdrop_screenshot_desktop.png
should be
"Input this key on another device,
or scan the QR code."

File: docs/docker-swarm-usage.md (hunk 2 above)
Comment (schlagmichdoch) [reply docs/docker-swarm-usage.md:17]: It actually seems to be uppercase `HEALTHCHECK` ([1](https://docs.docker.com/engine/reference/builder/#healthcheck))

**Comment by schlagmichdoch** [comment]
I might recreate the screenshots when I have implemented the public room feature. Thanks for the reminder!

File: docs/docker-swarm-usage.md (hunk 2 above)
Comment (schlagmichdoch) [reply docs/docker-swarm-usage.md:17]: @comradekingu I would love to merge this. Could you first resolve this please! :)

Comment (comradekingu) [reply docs/docker-swarm-usage.md]: Seems good now.

**Review by schlagmichdoch — APPROVED** [review]
Seems good now! Much cleaner and easier to read and understand. Thanks!

**Pull request merged**